### DISCUSSION TITLE
How to get MeteoFrance API KEY?

### DISCUSSION THREAD

**Discussion by dzungpv** [open-meteo/open-meteo#771] [Q&A] [answered]
Last year I can download the MeteoFrance model like: arpege_world, arpege_europe, arome_france and arome_france_hd.
But now it require API key mention here https://github.com/open-meteo/open-meteo/discussions/614.
But where to get the key? I emailed support.inspire@meteo.fr but they said that they does not manage these models

**Comment by patrick-zippenfenig** [comment, marked as answer]
The meteofrance API service is available here: http://portail-api.meteofrance.fr

**Reply by dzungpv** [reply]
I register account but the token only available with limited time. How do you deal with it?
